Jenkinsfile.quarkus

```
@@ -46,7 +41,7 @@ pipeline {
                 checkout([$class: 'GitSCM', branches: [[name: env.BRANCH_NAME]], browser: [$class: 'GithubWeb', repoUrl: 'git@github.com:kiegroup/kogito-apps.git'], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'kogito-apps']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'kie-ci-user-key', url: 'git@github.com:kiegroup/kogito-apps.git']]])
                 dir("kogito-apps") {
                     script {
-                      maven.runMavenWithSubmarineSettings('clean install -Dversion.io.quarkus=999-SNAPSHOT', false)
+                      maven.runMavenWithSubmarineSettings('clean install', false)
```

we may keep this and just use

```suggestion
                      catchError {
                          maven.runMavenWithSubmarineSettings('clean install', false)
                      }
```

so that it doesn't fail, and make it just unstable (IIRC) wydt?

---

I'm very sceptical about opening up the build like this, I feel that it can allow broken PRs to  get merged

---

ok just a suggestion :)